IdentityCore/src/requests/MSIDSilentTokenRequest.m

```diff
@@ -405,7 +405,7 @@ - (BOOL)isErrorRecoverableByUserInteraction:(NSError *)msidError
         return NO;
     }
     
-    MSIDErrorCode oauthError = MSIDErrorCodeForOAuthError(msidError.msidOauthError, MSIDErrorServerInvalidGrant);
+    MSIDErrorCode oauthError = MSIDErrorCodeForOAuthErrorWithSubErrorCode(msidError.msidOauthError, MSIDErrorServerInvalidGrant, msidError.msidSubError);
```

Comment:
Same here, is it needed in silent requests?

Reply:
It's in occurrence of replacing the "MSIDErrorCodeForOAuthError" with "MSIDErrorCodeForOAuthErrorWithSubErrorCode" as it falls back to the original implementation if nothing that specific scenario is not met

Reply:
Yes, but do you ever expect this scenario to happen in silent requests? If no, then no need to change silent request code.

Reply:
Okay, i will revert it back

Reply:
Thanks, will re-review after it's reverted and there're some unit tests.